content/logs/log_collection/_index.md

```diff
@@ -131,38 +109,76 @@ logs:
     include_units:
       - docker.service
       - sshd.service
 ```
-
 [9]: /agent/faq/agent-configuration-files/
 
 {{% /tab %}}
 {{% tab "Windows Events" %}}
 
-Set `type` to **windows_event** then specify the `channel_path` of your Windows channel.
+Edit the file `win32_event_log.d/conf.yaml` in the Agent's `conf.d/` directory (or in the Datadog Agent Manager) with the channels from which you want to send Windows Events as log. 
 
-Example: 
+To see the channel list run the following command in a powershell:
 
-[Refer to the dedicated widows event documentation page][15].
+```
+Get-WinEvent -ListLog *
+```
 
-[15]: https://docs.datadoghq.com/integrations/windows_event_log/#log-collection
+To see the most active ones run the following command:
 
-{{% /tab %}}
+```
+Get-WinEvent -ListLog * | sort RecordCount -Descending
+```
+
+Then add the channels in your `win32_event_log.d/conf.yaml` configuration file:
 
+```yaml
+logs:
+  - type: windows_event
+    channel_path: <CHANNEL_1>
+    source: <CHANNEL_1>
+    service: myservice
+    sourcecategory: windowsevent
+
+  - type: windows_event
+    channel_path: <CHANNEL_2>
+    source: <CHANNEL_2>
+    service: myservice
+    sourcecategory: windowsevent
+```
+
+Edit the `<CHANNEL_X>` parameters with the wanted channel to collect events from. Set the `source` to the channel name to benefit from the integration automatic processing Pipeline setup.
+
+Finally [Restart the agent][1]
+
+[1]: /agent/basic_agent_usage/windows/
+
+{{% /tab %}}
 {{< /tabs >}}
 
+List of all available parameters for log collection: 
+
+| Parameter        | Presence  | Description                                                                                                                                                                                                                                                                                                                |
+| ------           | -------   | ------                                                                                                                                                                                                                                                                                                                     |
+| `type`           | mandatory | Type of the log input source, it can be **tcp**, **udp**, **file**, **windows_event**, **docker**, or **journald**                                                                                                                                                                                                         |
+| `port`           | mandatory | If `type` is **tcp** or **udp**, port to listen log from.                                                                                                                                                                                                                                                                  |
```

Comment:
Basically this entire list of `**`'d strings should be `back-ticked`…

Reply:
No only the parameter are between backquotes, parameter values are in bold to be consistent with the API doc https://docs.datadoghq.com/api/?lang=python#post-an-event